docs/cookbook/extract_event_details.md

```diff
@@ -0,0 +1,34 @@
+This recipe demonstrates how to use the `outlines` library to extract structured event details from a text message.
+We will extract the title, location, and start date and time from messages like the following:
+
+```plaintext
+Hello Kitty, my grandmother will be here, I think it's better to postpone
+our appointment to review math lessons to next Monday at 2pm at the same
+place, 3 avenue des tanneurs, one hour will be enough see you 😘
+```
+
+Let see how to extract the event details from the message with the MLX
+library dedicated to  Apple Silicon processor  (M series).
+
+```python
+--8<-- "docs/cookbook/extract_event_details.py"
```

Comment:
Will that inline the content of the python script?

Reply:
Yes and that python script will be evaluated by the precommit hooks. 

the mkdoc.yml need also the `pymdownx.snippets:` config

Reply:
Nice, didn't know about this